styles/inoreader/catppuccin.user.css

```diff
@@ -69,309 +69,485 @@
+  a.text-color {
+    color: @text !important;
+  }
 
-    #tree_pane,
-    .tree_ad,
-    .article_full_contents,
-    .article_unreaded,
-    .search_empty_state_mask,
-    .library_wrapper_outer,
-    .teams_presentation_wrapper,
-    .article_subscribed,
-    .tr1,
-    .search_width_constraint {
-      background-color: @base;
-      color: @text;
-    }
+  .text-muted-color {
+    color: @subtext0 !important;
+  }
 
-    .st_usage_slider_outer {
-      background-color: @text;
-    }
+  #tree_pane,
+  #reader_pane,
+  #feeds_nav,
+  .tree_ad,
+  .article_full_contents,
+  .article_unreaded,
+  .search_empty_state_mask,
+  .library_wrapper_outer,
+  .teams_presentation_wrapper,
+  .article_subscribed,
+  .tr1,
+  .search_width_constraint,
+  .bg-white {
+    background-color: @base !important;
+    color: @text;
+  }
 
-    .st_usage_slider_inner {
-      background-color: @accent-color;
-    }
+  .st_usage_slider_outer {
+    background-color: @text;
+  }
 
-    #hint-pref {
-      fill: @accent-color;
-    }
+  .st_usage_slider_inner {
+    background-color: @accent-color
+  }
 
-    .preferences_main_button,
-    .preferences_billing_feature_main {
-      background-color: @mantle;
-      border-color: @surface0;
-    }
+  #tree_pane .parent_div_inner_selected .plus_img {
+    color: @accent-color;
+  }
 
-    .dashboard_gadgets,
-    .dashboard_gadgets .content,
-    .library_section_main_article,
-    .library_secondary_article,
-    .library_section_list_articles {
-      background-color: @mantle;
-    }
+  .active-url .item-name,
+  .active-url span[class^="icon-"].h4,
+  .active-url i[class^="icon-"].h4 {
+    color: @accent-color !important;
+  }
 
-    input[type="text"]:focus,
-    input[type="checkbox"]:focus + label,
-    input[type="checkbox"]:active + label,
-    input[type="button"]:focus,
-    input[type="button"]:active,
-    textarea:focus,
-    select:focus,
-    button:focus,
-    input[type="password"]:focus,
-    input[type="email"]:focus,
-    button:active {
-      box-shadow: 0 0 3px @accent-color !important;
-    }
+  #tree_pane .plus_img,
+  #tabs_pane .nav-item a i,
+  #tabs_pane #bottom_tabs .nav-item a i {
+    color: @text;
+  }
 
-    input[type="text"],
-    input[type="number"],
-    input[type="password"],
-    select,
-    textarea {
-      background-color: @base;
-      color: @text;
-      border-color: @subtext0;
-    }
+  #hint-pref {
+    fill: @accent-color;
+  }
 
-    .graylink_darker,
-    .article_short_contents {
-      color: @subtext0;
-    }
+  .article_footer_buttons.icon-article_topbar_read_later_full, #reader_pane .ar .article_btns a .h4.icon-saved,
+  #reader_pane .ar .article_btns a .h4.star_full,
+  span.icon-yellow,
+  i.icon-yellow {
+    color: @yellow;
+  }
 
-    div.article_magazine.article_current,
-    div.article_magazine.article_bulk_selected {
-      box-shadow: 0 0 0 1px @subtext0;
-    }
+  .bg-alt {
+    background-color: @yellow;
+    background: @yellow !important;
```

Comment:
why are you using both ``background-color`` and ``background``?

Reply:
The linter catches this. Please don't comment on it Omar.